### PR TITLE
Untracked/storybook

### PR DESCRIPTION
**Pull request by ziggabyte**
## Description
This PR adds a bunch of Storybook stories to our Zetkin... components.

## Changes
* Adds storybook stories for all Zetkin... components.

### CODE REVIEW

File: src/components/ZetkinDateTime.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,26 @@
...
+
+export const basic = Template.bind({});
+basic.args = {
+  datetime: new Date().toISOString().slice(0, 19),
+};
+
+export const convertToLocal = Template.bind({});
+convertToLocal.args = {
+  convertToLocal: true,
+  datetime: new Date().toISOString().slice(0, 19),
```
Comment (djbusstop) [inline src/components/ZetkinDateTime.stories.tsx:25]: I feel like there should be a built in method for this, or you can use `.toJson()`

File: src/components/ZetkinEllipsisMenu.stories.tsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+    {
+      label: 'Dirty Dancing',
+      subMenuItems: [
+        {
+          label: 'Dirty Dancing 2: Electric Boogaloo',
+        },
+      ],
+    },
+  ],
+};
```
Comment (djbusstop) [inline src/components/ZetkinEllipsisMenu.stories.tsx:42]: Nice!

File: src/components/ZetkinQuery.stories.tsx [hunk 3]
```diff
@@ -0,0 +1,47 @@
...
+
+export const customLoading = Template.bind({});
+customLoading.args = {
+  loadingIndicator: <LinearProgress />,
+  queries: { mockLoadingQuery },
+};
+
+export const error = Template.bind({});
+error.args = {
+  errorIndicator: <Typography>THERE WAS AN ERROR!!!</Typography>,
```
Comment (djbusstop) [inline src/components/ZetkinQuery.stories.tsx:39]: This is a custom error, there's also a default error. Maybe call this custom error? Or make one for default error

File: src/components/ZetkinRelativeTime.stories.tsx [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+export const basic = Template.bind({});
+basic.args = {
+  datetime: '2022-06-01T14:53:15',
+};
+
+export const toLocal = Template.bind({});
+toLocal.args = {
+  convertToLocal: true,
+  datetime: '2022-06-01T14:53:15',
+};
```
Comment (djbusstop) [inline src/components/ZetkinRelativeTime.stories.tsx:27]: Might be really nice to have a little documentation here about how and why this is used, or what impact it has. Storybook can generate some interactive docs im pretty sure.

File: src/components/ZetkinSpeedDial/ZetkinSpeedDial.stories.tsx [hunk 5]
```diff
@@ -0,0 +1,18 @@
...
+  title: 'Molecules/ZetkinSpeedDial',
+} as ComponentMeta<typeof ZetkinSpeedDial>;
+
+const Template: ComponentStory<typeof ZetkinSpeedDial> = (args) => (
+  <ZetkinSpeedDial actions={args.actions} />
+);
+
+export const basic = Template.bind({});
+basic.args = {
+  actions: [ACTIONS.CREATE_CAMPAIGN, ACTIONS.CREATE_EVENT, ACTIONS.CREATE_TASK],
```
Comment (djbusstop) [inline src/components/ZetkinSpeedDial/ZetkinSpeedDial.stories.tsx:17]: Maybe better to create a new action?

**Review by djbusstop — COMMENTED** [review]
Just some comments!

File: src/components/ZetkinSpeedDial/ZetkinSpeedDial.stories.tsx (hunk 5 above)
Comment (ziggabyte) [reply src/components/ZetkinSpeedDial/ZetkinSpeedDial.stories.tsx:17]: explain, i don't understand?

File: src/components/ZetkinDateTime.stories.tsx (hunk 1 above)
Comment (ziggabyte) [reply src/components/ZetkinDateTime.stories.tsx:25]: from my research now, the `toJSON()` still puts a Z at the end. The option people seem to do is either slicing out the Z, or doing `new Date().format('YYYY-MM-DD[T]HH:mm:ss')`. Which approach do you think makes more sense? I think either way I should add a comment of why the slicing or formatting is happening.

File: src/components/ZetkinRelativeTime.stories.tsx (hunk 4 above)
Comment (ziggabyte) [reply src/components/ZetkinRelativeTime.stories.tsx:27]: it seems in their docs that you're supposed to just be able to put comments in the stories-file and they will render in the docs tab in storybook, but I'm not getting it to work for me :(

File: src/components/ZetkinDateTime.stories.tsx (hunk 1 above)
Comment (djbusstop) [reply src/components/ZetkinDateTime.stories.tsx:25]: Oh really `toJson` puts a Z at the end woah. Ok, hm I say then stick with the slice and as you said, use a comment :)

File: src/components/ZetkinSpeedDial/ZetkinSpeedDial.stories.tsx (hunk 5 above)
Comment (djbusstop) [reply src/components/ZetkinSpeedDial/ZetkinSpeedDial.stories.tsx:17]: You're using the `ACTION.CREATE_CAMPAIGN` but could write a new action? It's just a suggestion, that could show in the code how actions are made.

File: src/components/ZetkinRelativeTime.stories.tsx (hunk 4 above)
Comment (djbusstop) [reply src/components/ZetkinRelativeTime.stories.tsx:27]: I'll look in to this!

File: src/components/ZetkinSpeedDial/ZetkinSpeedDial.stories.tsx (hunk 5 above)
Comment (ziggabyte) [reply src/components/ZetkinSpeedDial/ZetkinSpeedDial.stories.tsx:17]: aaah nice idea! Will do :)

**Review by djbusstop — DISMISSED** [review]
I agree with @richardolsson We can merge this and improve later :)

**Review by lowlandjuju — CHANGES_REQUESTED** [review]
Great effort in adding so many components - I think I've just found a couple of issues, and they're not ones that are related to the bits you've added, but maybe you could fix them so that this PR leaves us with a storybook that has no errors or things out of place?

- The "Example" section should be removed and the components inside relocated somewhere else
- The Organisms -> TextEditor story is broken, it probably can be fixed with something straightforward 
❤️

**Comment by ziggabyte** [comment]
> Great effort in adding so many components - I think I've just found a couple of issues, and they're not ones that are related to the bits you've added, but maybe you could fix them so that this PR leaves us with a storybook that has no errors or things out of place?
> 
>     * The "Example" section should be removed and the components inside relocated somewhere else
> 
>     * The Organisms -> TextEditor story is broken, it probably can be fixed with something straightforward
>       ❤️

This makes sense to me, not big changes i think, will look into them now and hopefully done in a min!

**Pull request merged**